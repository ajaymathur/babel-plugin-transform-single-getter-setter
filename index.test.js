const babel = require('babel-core');
const plugin = require('./');

var example = `
class Alert {
  static get VERSION() {
    return VERSION
  }
 }
`

it('matches snapshot', () => {
	const {code} = babel.transform(example, {plugins: [plugin]});
	expect(code).toMatchSnapshot();
});
