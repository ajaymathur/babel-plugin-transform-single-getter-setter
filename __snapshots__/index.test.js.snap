// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`works 1`] = `
"
class Alert {
  Object.defineProperty(Alert, \\"VERSION\\", {
    get: function () {
      return VERSION;
    },
    enumerable: true,
    configurable: true
  });
}"
`;
